.github/workflows/ci-core-reusable.yml

```diff
@@ -183,6 +184,7 @@ jobs:
           mkdir -p $SNAPSHOT_RECOVERY_LOGS_DIR
           mkdir -p $GENESIS_RECOVERY_LOGS_DIR
           mkdir -p $EXTERNAL_NODE_LOGS_DIR
+          mkdir -p $FEES_LOGS_DIR
```

Comment:
you forgot to add it to GITHUB_ENV

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2898/commits/9b363d6820b0be580889974f69f7cbbac6461e48